ArjunaJTA/object_store/pom.xml

```diff
@@ -47,6 +52,18 @@
       <artifactId>artemis-journal</artifactId>
       <version>2.18.0</version>
     </dependency>
+    <!-- JDBCStore dependency on h2 -->
+    <dependency>
+      <groupId>com.h2database</groupId>
+      <artifactId>h2</artifactId>
+      <version>${version.com.h2database}</version>
```

Comment:
would you use ${version.h2}?

Reply:
If we plan to harmonise our approach to versions of dependencies we should, where it makes sense to, use the same property names and the same versions. In this quickstart I followed the way we do it in the main repo: 
 https://github.com/jbosstm/narayana/blob/main/pom.xml#L189

Reply:
But I think one of us should start thinking about creating narayana boms. That would be a big change though and I don't know if it would risk our plans for 6.0.0.Final, what do others think (I know we briefly discussed it on one of the earlier PRs this year)?

Reply:
I think it is a good idea! +1

Reply:
Which one of us :-)

Reply:
I created the Jira https://issues.redhat.com/browse/JBTM-3735 and the zulip topic (https://narayana.zulipchat.com/#narrow/stream/323715-developers/topic/JBTM-3735.20Create.20narayana.20boms)